README.md

```diff
@@ -58,7 +58,7 @@ $ set TELOXIDE_TOKEN=<Your token here>
 $ $env:TELOXIDE_TOKEN=<Your token here>
 ```
 
- 4. Make sure that your Rust compiler is up to date (`teloxide` currently requires rustc at least version 1.64):
+ 4. Make sure that your Rust compiler is up-to-date (`teloxide` currently requires rustc at least version 1.64):
```

Comment:
Why is this necessary?

Reply:
Actually, "up-to-date" [seems right](https://writingexplained.org/up-to-date-hyphenated).

Reply:
You're right, it isn't

Reply:
Oh what a subtlety, ok...